app/git/utils.py

```diff
@@ -301,6 +336,21 @@ def get_github_emails(oauth_token):
     return emails
 
 
+def get_emails_master(username):
+    from dashboard.models import Profile
+    to_emails = []
+    to_profiles = Profile.objects.filter(handle__iexact=username)
+    if to_profiles.exists():
+        to_profile = to_profiles.first()
+        if to_profile.github_access_token:
+            to_emails = get_github_emails(to_profile.github_access_token)
```

Comment:
Why don't we just use the email we already know about from their `User.email`?

Reply:
Agree with @mbeacom this can create a bug. 
There are some email situations a little estrange in github api one is the private email, if you put your profile with hidden email the first call of the profile doesn't list the email, also many people have their company email in that json and not the personal one.

Reply:
this method is designed to get all the emails that we can find via github, so that we have somewhere to send the tip information. its important to get a list of the emails so we don't get support requests asking 'where is my tip?'

i agree that having some sort of ranking of which email would be preferred to use would be ideal.

Reply:
Is this used in optin to fill my profile email? My consideration is only if used in tip action, because It will prefer to send to any of my github registered emails rather than the one I fill in my gitcoin profile.